.github/ISSUE_TEMPLATE/question.md

```diff
@@ -1,10 +1,9 @@
 ---
 name: Question
 about: Ask your Question here
-title: ''
+title: ""
```

Comment:
How come you want to change the ISSUE_TEMPLATE files? They are exactly how Github creates them when you use the UI.

Reply:
'markdownlint` standards made all the Markdown changes in this PR. Also, other linters/formatters tools use double quotes too.

If we behave similarly to the standards used by the tools, users do not need to be bothered when participating and constantly delete these tiny changes in their PRs. It should be considered that many users use formatting tools, and what Github makes is not necessarily the same as what users use.